sdk/java/src/test/java/sawtooth/sdk/client/SigningTest.java

```diff
@@ -19,7 +19,6 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.lang.String;
-import java.math.BigInteger;
```

Comment:
Is this change relevant of for this PR, or should it at least be in a separate commit?

Reply:
It could be in a separate commit. I removed 2 unneeded imports.

Reply:
I think I am going to drop the removal of the imports from the PR.